sdk/deviceupdate/Azure.Iot.DeviceUpdate/README.md

```diff
@@ -0,0 +1,75 @@
+# Device Update for IoT Hub client library for .NET
+
+The library provides access to the Device Update for IoT Hub service that enables customers to publish updates for their IoT devices to the cloud, and then deploy these updates to their devices (approve updates to groups of devices managed and provisioned in IoT Hub). 
+
+  [Source code](https://github.com/Azure/azure-sdk-for-sdk/tree/master/sdk/deviceupdate/Azure.Iot.DeviceUpdate/src/) | [Package](https://www.nuget.org/packages/Azure.DeviceUpdate) | [Product documentation](https://github.com/Azure/adu-private-preview/blob/master/docs/adu-overview.md)
```

Comment:
Linked doc pages like (https://github.com/Azure/adu-private-preview/blob/master/docs/adu-overview.md) are not public we need to remove them or link to public documentaiton.